README.md

```diff
@@ -9,45 +9,43 @@ BTP Manager is an operator for [SAP BTP Service Operator](https://github.com/SAP
 - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
 - Kubernetes cluster (you can use [k3d](https://k3d.io)) 
 
-## Installation
-Use the following commands to run BTP Manager locally. All `make` commands refer to [Makefile](./Makefile) in the `operator` directory.
+## Local installation
+Use the following commands to run BTP Manager controller from your host. All `make` commands refer to [Makefile](./Makefile) in the `operator` directory.
```

Comment:
If you are talking about the two commands only, it would be better to say 'Both `make` commands' instead of 'All the `make` commands'.